src/acp.js

```diff
@@ -91,14 +91,26 @@ function handleGlobalBanDelete(user, data) {
 }
 
 function handleListUsers(user, data) {
-    var name = data.name;
-    if (typeof name !== "string") {
-        name = "";
+    var query = data.query;
```

Comment:
"query" might be a confusing name for this, because it's overloaded with the concept of an SQL "query".  I might call it "searchTerm" or something like that.